pkgs/run-sotest/default.nix

```diff
@@ -23,7 +23,7 @@ python3Packages.buildPythonApplication rec {
     '';
 
     homepage = "https://sotest.io/";
-    license = licenses.unfree;
+    license = licenses.bsd3;
```

Comment:
3-clause BSD here disagrees with the MIT license file in the top-level directory.